lnwallet/channel.go

```diff
@@ -2704,6 +2704,14 @@ func (v *HtlcView) AuxTheirUpdates() []AuxHtlcDescriptor {
 	return fn.Map(v.Updates.Remote, newAuxHtlcDescriptor)
 }
 
+// FetchLatestHTLCView returns the latest HTLC view of the lightning channel.
+// The latest log indices are used for both the local and remote update logs.
```

Comment:
I think we need to expand a bit on this comment. What does it mean that we're using the latest log indices?
IIUC it means we include any HTLCs that might not yet have been committed or ACKed by both parties.
Which means to me that we probably need to be careful when using `ComputeView` on the `tapd` side, because there we're filtering HTLCs by a specific commitment TX height.